src/erlcloud_aws.erl

```diff
@@ -144,6 +144,11 @@ aws_request4(Method, Protocol, Host, Port, Path, Params, Service, Config) ->
             {error, Reason}
     end.
 
+aws_request4_no_update(put, Protocol, Host, Port, Path, Params, Service, #aws_config{} = Config) ->
```

Comment:
i would rather ask to add a `put` case clause at https://github.com/erlcloud/erlcloud/pull/289/files#diff-b6a88197b95cfa7d84db56a756107eadR156 five lines below

or be simpler and use alike: 
https://github.com/alertlogic/erlcloud/blob/master/src/erlcloud_aws.erl#L153-L161